bk-api/bk_api.py

```diff
@@ -585,17 +585,25 @@ def scp(source, node_id, target):
 
 
 
-def node_ssh(node_id, command, input_str=None):
+def node_ssh(node_id, command, input_str=None,quiet_mode=False):
```

Comment:
please run "linting" on your changes  (and maybe the whole file) to ensure normalization of white spaces and tabs.

i.e. there is no space here: `input_str=None,quiet_mode=False`

Reply:
If it is just a whitespace, I will fix it but turning on pylint will complain about everything. Are you guys using a specific config file for pylint?

Reply:
i use the VSCode linter: https://marketplace.visualstudio.com/items?itemName=ms-python.python

On save it formats the entire file.  What i tend to do is make 1 commit that is an auto-lint of the entire file. then i make my changes on top of that.

Reply:
Its really your call here.. but if we all use the same linting techniques our code will be ultra-pretty and look the same.

Reply:
Sounds good. The other way we can do it is in the CI to run a linter with a specific style guide so anything that is added to main/master will be abiding a specific style guide.